src/init/ossec-client.sh

```diff
@@ -235,7 +235,8 @@ stopa()
 
             if ! wait_pid $pid
             then
-                echo "Process ${i} couldn't be killed.";
+                echo "Process ${i} couldn't be killed. This process will be kill.";
```

Comment:
"kill" → "killed".

In addition, the complete sentence makes no sense:
> Process X couldn't be killed. This process will be killed.

How is the script going to kill a process that cannot be killed?

What do you think of this?
> Process X couldn't be terminated. It will be killed.